classes/fontproof.lua

```diff
@@ -57,11 +57,22 @@ function fontproof:init ()
   return plain.init(self)
 end
 
+local function getGitCommit()
+  -- If we are in a git repository, report the latest commit ID
+  local fh = io.popen("git describe --always --dirty='*'")
```

Comment:
```suggestion
  local fh = io.popen("git describe --long --tags --abbrev=7 --always --dirty='*'")
```
In most cases this will be the same result but if people use tags it will identify relative to those which I find quite useful. Also avoid issues with Git environments (usually ones setup by non-CLI tooling) that include wacky defaults